bedrock/contentful/api.py

```diff
@@ -23,10 +23,24 @@
 )
 from lib.l10n_utils import get_locale, render_to_string
 
-# Bedrock to Contentful locale map
-LOCALE_MAP = {
+# Bedrock and Contentful have slightly differing locale lists.
+BEDROCK_TO_CONTENTFUL_LOCALE_MAP = {
```

Comment:
Ah, I'll expand the code comment here. These are the mappings where the Contentful code and the Bedrock code don't directly match, so we need to fix them, rather than this being the _only_ ones we support. eg Bedrock has `pl` but Contentful models that as `pl-PL` so we 'flatten' it to match what Bedrock currently uses internally. 

Looking back at the list, the majority are in the pattern of the suffix just being the uppercase version of the prefix. 

`ms-MY` has been mapped to `ms` but looking at it cold, we should double-check that internally we are using that as Malaysian Malay and not anything to do with Monserrat.

Reply:
Looking at `ms-MY` <-> `ms` - this does make sense as the correct locale transformation (`ms` -> Malaysian Malay in production whereas`my` -> Burmese, for Myanmar)